CMakeLists.txt

```diff
@@ -514,9 +514,24 @@ target_compile_definitions(tvm_libinfo_objs PUBLIC DMLC_USE_LOGGING_LIBRARY=<tvm
 target_compile_definitions(tvm PUBLIC DMLC_USE_LOGGING_LIBRARY=<tvm/runtime/logging.h>)
 target_compile_definitions(tvm_runtime PUBLIC DMLC_USE_LOGGING_LIBRARY=<tvm/runtime/logging.h>)
 
+
+include(cmake/modules/Hexagon.cmake)
```

Comment:
You cannot move this include here.

Reply:
Why not?

Reply:
Because it treats Hexagon different from other targets---see my other comment for more details.